cpsk/__init__.py

```diff
@@ -50,7 +50,7 @@ def __repr__(self):
                                        self.distance)
 
 
-def get_routes(departure, dest, vehicle='vlakbus', time='', date=''):
+def get_routes(departure, dest, vehicle='vlakbus', time='', date='', direct='false'):
```

Comment:
Could you change `direct` keyword to be boolean and convert it to string afterwards?
```suggestion
def get_routes(departure, dest, vehicle='vlakbus', time='', date='', direct=False):
```